configs/bllFlow.json

```diff
@@ -1,14 +1,14 @@
 {
-  "index_name": "bllFlow",
+  "index_name": "bllflow",
```

Comment:
You can’t change it this way. We will have to provide you new credentials

Reply:
We'd appreciate the new credentials. Do we make a request by email? Or do you send them to the original email (they were originally sent to me).

Reply:
Please apply and reply with a link to this thread. I will handle it asap when I am back from PTO in 2 weeks